netbox_diode_plugin/api/common.py

```diff
@@ -96,6 +97,7 @@ def to_dict(self) -> dict:
             "id": self.id,
             "changes": [change.to_dict() for change in self.changes],
             "branch": self.branch,
+            "related_objects": [change.to_dict() for change in self.related_objects],
```

Comment:
What's idea behind `related_objects`? It doesn't seem to be set yet.

Reply:
oops yeah that's not meant to be in there hah.